docs/features/deserializing-events.md

```diff
@@ -5,6 +5,8 @@ layout: default
 
```

Comment:
I would provide a brief introduction on what this doc is and the supported ways to deserialize events with a pointer to the doc section (list-approach for example)

Reply:
Added one.